.automation.conf/config.sh

```diff
@@ -3,7 +3,7 @@
 
 # See: https://github.com/stackhpc/docker-rally/blob/master/bin/rally-verify-wrapper.sh for a full list of tempest parameters that can be overriden.
 # You can override tempest parameters like so:
-export TEMPEST_CONCURRENCY=8
+export TEMPEST_CONCURRENCY=6
```

Comment:
Need to bear in mind this is a shared file that affects all environments.

Reply:
Could we not use conditional statements to test the `KAYOBE_ENVIRONMENT`? 

if `ci-multinode` 6 else 8.

Reply:
Looks like the original value was 2